mysql-test/suite/funcs_1/t/is_check_constraints.test

```diff
@@ -0,0 +1,69 @@
+--source include/have_innodb.inc
+--echo #
+--echo # MDEV-14474: Create INFORMATION_SCHEMA.CHECK_CONSTRAINTS
+--echo #
+
+set check_constraint_checks=1;
+
+use test;
+create table t0
+(
+ t int, check (t>32) # table constraint
+) ENGINE=myisam;
+
+--vertical_results
```

Comment:
Please remove the vertical_results and horizontal_results modifiers. It doesn't improve readability in this case and it makes it harder to track.